.gitignore

```diff
@@ -20,3 +20,4 @@ yarn-error.log
 config/local.js
 
 .DS_Store
+.idea
```

Comment:
Did you mean to commit this? This is because Webstorm, right? Consider https://gist.github.com/subfuzion/db7f57fff2fb6998a16c instead.

Reply:
Woah, how have I never heard of that?! Nice!